ccd/templates/definition-importer.yaml

```
@@ -26,7 +26,7 @@ spec:
           flexVolume:
             driver: "azure/kv"
             secretRef:
-              name: kvcreds 
+              name: {{ default "kvcreds" .Values.importer.kvSecretRef }} 
```

the filter + default looks over kill?

shouldn't this just be?
```suggestion
              name: {{ .Values.importer.kvSecretRef }} 
```

---

Better safe than sorry ;-)